algolia/api-clients-automation
# fix(scripts): use glob to apply wildcard before rm

## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

- use glob to apply wildcard and get a list of files to remove
- fix error message in `check.yml`

## 🧪 Test
Tested locally by following this path:
- in `AlgolaSwiftGenerator`, change the destination file name for the `client_configuration.mustache` template to `ConfigurationClient.swift`
- in `removeExistingCodegen.ts`, change the wildcard to `*Client.swift`
- run `apicb generate swift attesting -v`
- confirm that the script shows it will remove only the `AbtestingClient.swift` file and the `Models` folder
- confirm that the old `Configuration.swift` file is still there
- fix everything
- ???
- profit!
